### DISCUSSION TITLE
connected glass plz  :)

### DISCUSSION THREAD

**Discussion by xiaoqiangisaminecraftfan** [IrisShaders/Iris#310] [Q&A] [answered]
iris is a good mod,it helped minecraft better,alot of things is better than optifine.But,the connected galss is what i want
It let the building look so much better,the normal glass has some border , that is a big trouble for us 
So,i hope we can get a better galss for iris.If it relly does,i'll be so much like it.
And thank u help minecraft be more good
Hope you have a greatful day !  :)
If the connected glassis my contrl trouble please tell me
If it can't be connected,that's OK,i will still not be sad:)

if it relly got one.I will show it to my friend,
i'm relly sure they will be so supurisd
And i;m a English green hand,if my word have some poblum,plz,Don't care them

**Comment by xiaoqiangisaminecraftfan** [comment]
i saw a lot of galss connected mod or somrthings but i can't see even one is not need optifine,is it a optfine owner?or what.Yesterday,i go to looking for some glass connected,5fours later....Nothing...

**Comment by DragonEggBedrockBreaking** [comment]
There is a seperate mod for fabric that adds connected glass: https://github.com/DragonEggBedrockBreaking/connected-block-textures/releases/latest

**Reply by xiaoqiangisaminecraftfan** [reply]
oh i will go to see,thank for your answer

**Reply by xiaoqiangisaminecraftfan** [reply]
it's a jar,isn't need to be a ".ZIP"?

**Reply by xiaoqiangisaminecraftfan** [reply]
oh .i see

**Reply by xiaoqiangisaminecraftfan** [reply]
no it didn't let galss connected

**Reply by xiaoqiangisaminecraftfan** [reply]
os ot need a shader or a rexturespack?

**Reply by DragonEggBedrockBreaking** [reply]
> it's a jar,isn't need to be a ".ZIP"?

All mods are a .jar file

**Reply by DragonEggBedrockBreaking** [reply]
> no it didn't let galss connected

You need to put it into your `mods` folder.

**Reply by xiaoqiangisaminecraftfan** [reply]
yeah,i put it in te mods,but,i see it can't be together with sodium?

**Reply by xiaoqiangisaminecraftfan** [reply]
i use iris becuse it can bo together with sodium,if this can't........sorry,i can't use it

**Reply by DragonEggBedrockBreaking** [reply]
you need another mod to make this work with sodium

**Reply by DragonEggBedrockBreaking** [reply]
https://github.com/comp500/Indium

**Reply by DragonEggBedrockBreaking** [reply]
since you use iris, i'm guessing you know how to build indium yourself

**Reply by xiaoqiangisaminecraftfan** [reply]
help me plz，ican‘t，i use iris is becuse,someone help me to made it

**Reply by xiaoqiangisaminecraftfan** [reply]
i don't konw how to build it,i'm so suck

**Comment by xiaoqiangisaminecraftfan** [comment]
Sorry,i can't build a mod,i use iris becuse somebody help me   awa

**Comment by NoComment1105** [comment, marked as answer]
An optifine format mod called [Continuity](https://modrinth.com/mod/continuity) that does just this

**Reply by marcellonovak** [reply]
There's no forge version

**Reply by DragonEggBedrockBreaking** [reply]
> There's no forge version

There is also no official forge version of iris. If you would like connected glass on forge with iris, ask the developers of an unofficial port.
